analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/VerifierBuilder.cs

```diff
@@ -46,6 +46,8 @@ internal record VerifierBuilder
         public string CodeFixTitle { get; init; }
         public bool ConcurrentAnalysis { get; init; } = true;
         public CompilationErrorBehavior ErrorBehavior { get; init; } = CompilationErrorBehavior.Default;
+        public string Framework { get; init; } = "net7.0";
+        public bool IsRazor { get; init; }
```

Comment:
`Framework` doesn't make much sense for normal rule UTs. Wouldn't `RazorVerifierBuilder`  and `RazorVerifier` make more sense here?

Reply:
I agree that it might cause some confusion, what about `RazorFramework`?

Reply:
Did you mean to create another record named `RazorVerifierBuilder`? If that's the case isn't it a bit an overkill to just store a single property?

Reply:
RazorFramework sound good. And then it could stay here, like the codefix functionality